CIPs/0020.md

```diff
@@ -0,0 +1,167 @@
+# CIP [0020]: Generic hash function support via extensible precompile
```

Comment:
I'd remove this line just to ensure the preamble remains at the beginning of the line. That way when we have a CIP site, it can scrape easily from the preamble.